src/express/index.js

```diff
@@ -1,24 +1,33 @@
 'use strict';
 
+const path = require('path');
 const express = require(`express`);
+const homeRoute = require('./routes/homeRoute');
 const searchRoutes = require(`./routes/search`);
 const myRoutes = require(`./routes/my`);
 const loginRoutes = require(`./routes/login`);
 const offersRoutes = require(`./routes/offers`);
 const registerRoutes = require(`./routes/register`);
+const errorsRoutes = require(`./routes/errors`);
+
+const STATIC_DIR = path.join(__dirname, `./../../markup`);
 
 const app = express();
 const port = 8080;
 
-app.get(`/`, (req, res) => {
-  res.send(`/`);
-});
 
+app.set(`view engine`, `pug`);
+app.set(`views`, path.join(__dirname, `./templates`));
+
+app.use(`/`, homeRoute);
```

Comment:
По аналогии с другими рутами 
```suggestion
app.use(`/`, homeRoutes);
```